go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/openshift/api v3.9.0+incompatible
 	github.com/openshift/client-go v0.0.0-20201020082437-7737f16e53fc
 	github.com/vmware/alb-sdk v0.0.0-20230628102904-861ce312a6bf
-	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes v0.0.0-20220405050344-3a6d72bbda3e
+	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes v0.0.0-20230907055619-402bb2efba83
```

Comment:
which release of AKO is this? 1.10.1 or 1.11.1?
I think we should target 1.10.1 for now and 1.11.1 later.